configuration/archive.md

```diff
@@ -153,3 +153,82 @@ These are the available parameters for the `Cloud-Archive` configuration block.
 * `Insecure-Skip-TLS-Verify` (optional): specifies that the client will ignore invalid TLS certificates when connecting to the Cloud-Archive server, defaults to false.
 * `Insecure-Disable-TLS` (optional): specifies that the client will use a cleartext connection when communicating with the Cloud-Archive server, defaults to false.
 
+### Backend Processors
+
+#### File Backend Type
+
+The File backend type is the default Cloud Archive backend plugin, if no `Backend-Type` is specified in the `[Global]` section then the `file` type is assumed.  The `file` driver is designed to store raw Gravwell shards organized in an way that allows rapid re-import or direct binding by a Gravwell instance.  Use the file type if your storage array can support seek operations and you wish to be able to directly bind a Gravwell instance to the storage location and perform queries without thawing or re-ingesting the data.
+
+##### Required Configuration Parameters
+
+* `Storage-Directory`: Directory must be writable and POSIX compliant.
+
+#### FTP Backend Type
+
+The FTP backend type is an optional variation of the `file` backend type that directly stores shards on a remote FTP server rather than local file storage.  This backend type can be useful as a method to access some tape storage systems that are frontended by FTP protocols.  The FTP backend type does not modify shards and could potentially enable direct binding to data without thawing depending on the underlying storage architecture.
+
+##### Required Configuration Parameters
+
+* `Backend-Type`: Must be configured as "ftp" to enable the FTP backend.
+* `FTP-Server`
+* `FTP-User`: May be "anonymous" for unauthenticated access.
+* `FTP-Password`: May be empty when using "anonymous" for unauthenticated access.
+
+#### S3 Backend Type
+
+The S3 backend type is designed for system agnostic, low cost, long term storage of archive data. S3 backed storage is re-encoded and compressed into smaller chunks and then uploaded to S3-compatible storage, shards sent to S3 cannot be directly bound to by a Gravwell instance and must be re-thawed for query.  The data formats are open and designed to support alternate query data access methods like AWS S3 Select.
+
+##### Required Configuration Parameters
+
+* `Backend-Type`: Must be configured as "ftp" to enable the FTP backend.
```

Comment:
It made sense to me to have this description for the Backend-Type in the FTP Backend Type section, but should this have an S3-specific description in this section?